rules/civilian-props.yaml

```diff
@@ -102,29 +102,467 @@ camsc13:
 	Armor:
 		Type: Concrete
 
-calit01e:
+ammocrat:
 	Inherits: ^CivBuilding
 	Tooltip:
-		Name: Light
+		Name: Ammo Crates
+	Armor:
+		Type: wood
 	Health:
-		HP: 50
+		HP: 1
+	RenderSprites:
+		Image: ammo01
+
+casin01e:
+	Inherits: ^CivBuilding
+	Tooltip:
+		Name: Roadblock
 	Armor:
-		Type: Steel
+		Type: concrete
+	Health:
+		HP: 5
+	Building:
```

Comment:
This shouldn't be necessary as it is inherited from `^CivBuilding`which inherits this from `^Building`.
(Same for the actors below that inherit `^CivBuilding`.)